### ISSUE TITLE
Write README.md

### ISSUE THREAD

**Issue by secretdeveloperisme** [tienphuckx/anony-box-backbone#14]

**Comment by tienphuckx** [comment]
It would be great if you can add more some info for each issue that make our team easy to understand the problem

**Comment by secretdeveloperisme** [comment]
> It would be great if you can add more some info for each issue that make our team easy to understand the problem

Yes, that is a good idea so I will add related information to the next issues. thanks
